source/jormungandr/jormungandr/scenarios/helper_classes/fallback_durations.py

```diff
@@ -106,13 +106,15 @@ def _get_duration(self, resp, place):
     @new_relic.distributedEvent("routing_matrix", "street_network")
     def _get_street_network_routing_matrix(self, origins, destinations):
         try:
-            return self._instance.get_street_network_routing_matrix(
+            if not self._streetnetwork_service:
```

Comment:
this case can happen? if yes  we should raise an exception as it look more like an error don't you think?

Reply:
I should throw a `TechnicalError` in `StreetNetworkBackendManager::get_street_network_db` if it is `None` in that case. Hence this line will be useless. Thanks!